pageserver/src/bin/pageserver.rs

```diff
@@ -256,6 +258,23 @@ fn start_pageserver(conf: &'static PageServerConf) -> anyhow::Result<()> {
     };
     info!("Using auth: {:#?}", conf.auth_type);
 
+    match var("ZENITH_AUTH_TOKEN") {
```

Comment:
Random thought, can we rename `ZENITH_AUTH_TOKEN` to `NEON_AUTH_TOKEN`? We don't use it extensively yet, should be easy to rename in all places.